### PR TITLE
Closes #6

### LINKED ISSUE #6: Cats vs Dogs? 🐶  🐱
Our client has asked us to decide the ultimate question - cats or dogs? We can't possibly choose one, so we thought - why not both?

- [] Open `index.html`
- [] Grab a placeholder image from [PlaceKitten](https://placekitten.com/) and add it to the index.html page. You can choose the size.
- [] Grab a placeholder image from a Google image search of puppies. Add the puppy photo to our repo to the `img` directory. Link to that image in the `index.html` page.
- [] Merge the file into master

Please submit your PR with a checklist, showing that you have done these three things. 

BONUS: Squash your commits.

### PR DESCRIPTION
**Pull request by merylturner**
- [x] Open index.html
- [x] Grab a placeholder image from PlaceKitten and add it to the index.html page. You can choose the size.
- [x] Grab a placeholder image from a Google image search of puppies. Add the puppy photo to our repo to the img directory. Link to that image in the index.html page.
- [x] Merge the file into master

### CODE REVIEW

**Comment by VerteDinde** [comment]
Thanks Meryl! @shanselman, would you mind reviewing?

**Review by shanselman — COMMENTED** [review]
Did you get a license for this puppy?

File: index.html [hunk 1]
```diff
@@ -21,6 +21,8 @@
 
         <!-- Add your site or application content here -->
         <p>Hello world! This is HTML5 Boilerplate.</p>
+        <img src="http://placekitten.com.s3.amazonaws.com/homepage-samples/200/140.jpg" width="400" height="280" alt="kitten"/>
+        <img src="img/cute_puppy.jpg" width="680" height="510" alt="puppy"/>
```
Comment (shanselman) [inline index.html:25]: Is this a Creative Commons Puppy? Check search.creativecommons.org for appropriately licensed puppies.

**Review by shanselman — APPROVED** [review]

**Comment by shanselman** [comment]
:shipit:

**Pull request merged**